api/lorawan.proto

```diff
@@ -255,7 +255,7 @@ message TxSettings {
   // Frequency (Hz).
   uint64 frequency = 4;
   // Transmission power (dBm). Only on downlink.
-  int32 tx_power = 5;
+  float tx_power = 5;
```

Comment:
This only breaks to gRPC gateway clients, which don't exist for V3 yet, and this is used in the NS-GS interface.